v2/tools/generator/internal/astbuilder/builder.go

```diff
@@ -367,9 +367,16 @@ func WrappedError(errorsPackage string, str string) dst.Expr {
 //
 // <pkg>.<name>
 func QualifiedTypeName(pkg string, name string) *dst.SelectorExpr {
+	return QualifiedName(pkg, name)
+}
+
+// QualifiedName generates a selector based on an identifier
+//
+// <pkg>.<name>
+func QualifiedName[S ~string](pkg string, name S) *dst.SelectorExpr {
```

Comment:
Using generics here allows us to skip casting `PropertyName` to a string explicitly when calling this.